### ISSUE TITLE
Params from checkbox 

### ISSUE THREAD

**Issue by araslanov-e** [igor-alexandrov/wiselinks#71]
```
check_box("post", "validated")
# => <input name="post[validated]" type="hidden" value="0" />
#    <input type="checkbox" id="post_validated" name="post[validated]" value="1" />
```

Если это поле будет активным, то в запрос попадет 
`post[validated]=0,1` вместо `post[validated]=1` 

[show Github](https://github.com/igor-alexandrov/wiselinks/blob/master/lib/assets/javascripts/_form.js.coffee#L35)

Правильно ли это?

**Comment by igor-alexandrov** [comment]
Нет, не правильно. Нужно подумать, как обрабатывать подобные случаи.

**Comment by araslanov-e** [comment]
А для каких случаев используются значения через запятую?
В случае со списком чекбоксов, в параметре должен быть массив значений
т.е. `post[validated][]=1&post[validated][]=2`
